src/ua/shalypenko/JDBC/JDBCMethods.java

```diff
@@ -0,0 +1,155 @@
+package ua.shalypenko.JDBC;
+
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+public class JDBCMethods {
+
+    public static String input(String message) {
+        Scanner scan = new Scanner(System.in);
+        System.out.println(message);
+        return scan.nextLine();
+    }
+
+    public static void insertIntoTableContact() {

```

Comment:
N: When you insert, maybe you want to continue work with some entity.
So this method should return smth